Source/WebCore/crypto/keys/CryptoKeyEC.h

```diff
@@ -33,14 +33,16 @@
 
 #if OS(DARWIN) && !PLATFORM(GTK)
 #include "CommonCryptoUtilities.h"
+#include <wtf/Vector.h>
+
+using PlatformECKey = std::variant<CCECCryptorRef, Vector<uint8_t>>;
```

Comment:
Since CCECCryptorRef is just a ref, I wonder whether we should try to return :
`std::variant<CCECCryptorRef, std::reference_wrapper<Vector<uint8_t>>>`.
That will remove the memory allocation/copy in case of variant be a vector.

Another approach would be to remove platformKey() and introduce platformKeyAsCCECCryptoRef() and platformKeyAsByteArray().
These getters would return either a `CCECCryptorRef` or a `const Vector<uint8_t>&`